ztp/gitops-subscriptions/argocd/example/policygentemplates/common-ranGen.yaml

```diff
@@ -76,6 +76,18 @@ spec:
     # - fileName: OadpOperatorStatus.yaml
     #   policyName: "subscriptions-policy"
     #
+    # NMState operator is used for IPsec configuration with NMState
```

Comment:
Maybe note that it's only recommended for multi-node deployments?

Reply:
I believe this is now resolved in the line below